webinar-flow/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: "3"
+services:
+  mongi:
```

Comment:
Un authenticated database. Anyone can hit 27017 directly

Reply:
I deleted .gitignore by mistake thats why by mistake .env file got pushed. Should I delete this pull request and send a new pull request?

Reply:
Change the password of the mongoDB cluster

Reply:
Yes I have told Subrah he said he will change it